prusti-specs/src/export_spec_rewriter.rs

```diff
@@ -0,0 +1,336 @@
+use proc_macro2::{TokenStream, TokenTree};
+use quote::{quote, quote_spanned, ToTokens};
+use syn::spanned::Spanned;
+
+// generate a hash of path
```

Comment:
Please use triple slashes `///` for comments that are actually meant to document what a method does.